products/google-nexus.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Google Nexus
+category: device
+tags: google mobile-phone tablet
+iconSlug: google
+permalink: /google-nexus
+versionCommand: "Settings -> About Phone -> Regulatory labels"
+releasePolicyLink: https://support.google.com/nexus/answer/4457705
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: true
+discontinuedColumn: true
+eolColumn: Guaranteed Software Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android # https://en.wikipedia.org/wiki/Google_Nexus
+    description: Supported Android versions range
+    link: https://endoflife.date/android
+
+# Supported Android versions range is based on https://www.gsmarena.com/.
+releases:
```

Comment:
Entries must be ordered by their release date.

Reply:
I already ordered them by their release date.

Reply:
Yes, but the order must be reversed (from the newest to the oldest). Sorry I should have mentioned that.